### PR TITLE
PMM-4959 Add stage to release OVF image.

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: pmm/pmm2-server-release.groovy [hunk 1]
```diff
@@ -217,6 +221,38 @@ pipeline {
...
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                    sh """
+                        aws s3 cp --only-show-errors s3://percona-vm/\${OVF_VERSION} pmm2-server-\${VERSION}.ova
+                    """
+                }
+                sh """
+                    ssh -i ~/.ssh/id_rsa_downloads -p 2222 jenkins@jenkins-deploy.jenkins-deploy.web.r.int.percona.com "mkdir -p /data/downloads/pmm2/\${VERSION}/ova" || true
+                    sha256sum pmm2-server-\${VERSION}.ova > pmm2-server-\${VERSION}.sha256sum
```
Comment (hors) [inline pmm/pmm2-server-release.groovy]: @AlekSi  Do we need to use "pmm2-server-${VERSION}" or "pmm-server-${VERSION}" as a name for ovf image ?

File: pmm/pmm2-server-release.groovy [hunk 2]
```diff
@@ -19,6 +19,10 @@ pipeline {
             defaultValue: 'perconalab/pmm-server:dev-latest',
             description: 'pmm-server container version (image-name:version-tag)',
             name: 'DOCKER_VERSION')
+        string(
+            defaultValue: '',
+            description: 'OVA image filename',
+            name: 'OVF_VERSION')
```
Comment (AlekSi) [inline pmm/pmm2-server-release.groovy]: Why call it "version" if description above clarifies it is not a version?

File: pmm/pmm2-server-release.groovy (hunk 1 above)
Comment (AlekSi) [reply pmm/pmm2-server-release.groovy]: For Docker we use (at https://www.percona.com/downloads/pmm2/) `pmm-server-2.1.0`, so I guess we should do the same for OVF.
@rnovikovP ?

File: pmm/pmm2-server-release.groovy [hunk 3]
```diff
@@ -19,6 +19,10 @@ pipeline {
             defaultValue: 'perconalab/pmm-server:dev-latest',
             description: 'pmm-server container version (image-name:version-tag)',
             name: 'DOCKER_VERSION')
+        string(
+            defaultValue: '',
+            description: 'OVA image filename',
```
Comment (puneet0191) [inline pmm/pmm2-server-release.groovy]: Image Tag?

File: pmm/pmm2-server-release.groovy (hunk 1 above)
Comment (rnovikovP) [reply pmm/pmm2-server-release.groovy]: Yes. Pmm2-client was the only one place where we changed the name

File: pmm/pmm2-server-release.groovy (hunk 3 above)
Comment (hors) [reply pmm/pmm2-server-release.groovy]: @puneet0191 I can not say that it is ova image tag, we specify image name but not a tag.

**Pull request merged**